app/Exceptions/Auth/ApiException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Exceptions\Auth;
```

Comment:
Will `ApiException` be used only for auth actions?

The current name suggests that this is some kind of basic exception for all API actions, so shouldn't it be placed in `Brewmap\Exceptions` namespace or renamed to `AuthException`?